src/user/proposals/UserProposalDashboard/DashboardContent/DashboardContent.js

```diff
@@ -108,19 +126,29 @@ class DashboardContent extends Component {
         });
         this.setState({
```

Comment:
@AuraOfDivinity please use it as a function and pass these values in params, it will reduce the code